product_docs/docs/epas/15/epas_compat_sql/21_create_public_database_link.mdx

```diff
@@ -325,6 +325,43 @@ Pushdown applies to statements with the standard SQL join operations (inner join
 
 In order for pushdown to occur, certain basic conditions must be met. The tables involved in the join operation must belong to the same foreign server and use the identical connection information to the foreign server (that is, the same database link defined with the `CREATE DATABASE LINK` command).
 
+The version 15 and later provides the `edb_dbLink_oci_pushdown.config` configuration file to restrict the pushdowns. You can define the list of functions and operators in this file that can pushdown to the remote server. You can easily add or modify the list as per the requirements.
```

Comment:
```suggestion
The version 15 and later provides the `edb_dbLink_oci_pushdown.config` configuration file to restrict pushdowns. You can define the list of functions and operators in this file that can push down to the remote server. You can easily add or modify the list as per your requirements.
```
Again I wonder if we need to say this is a v15 feature when this is in the v15 version of the docs.

Reply:
v15 is removed